joomla-projects/gsoc22_accessibility-plugin-script
# Implemented Speech velocity

### Summary of changes
1. Users can change the speech velocity.
2. Made a11y accessible with keyboard.
3. fixed : https://github.com/joomla/joomla-cms/issues/30598

### Testing instructions

1. Make sure you can change the speech velocity.
2. check you can access a11y with keyboard.
3. Make sure you can click links and buttons when screen reader is enabled.
